### PR TITLE
CCXDEV-7974: show error message when Insights Operator cannot process results

### PR DESCRIPTION
**Pull request by gkarat**
Implements https://issues.redhat.com/browse/CCXDEV-7974.
Mocked in https://marvelapp.com/prototype/g70a1he/screen/82052851.

**Changes:**
- public: adds a default error state component (used in console.redhat.com) with the danger icon and links to Red Hat Support and status pages.
- Insights Advisor widget: adds additional state to the Insights Advisor component and renders an error message if there `Degraded` and `UploadDegraded` conditions are set to true (= is an outage and Insights Operator cannot send archives and process Insights results).
- Insights Advisor widget: refactors the Prometheus metric: return all conditions instead of only Disabled.

**This state can be triggered by running**
```
oc create secret generic support --from-literal=endpoint=https://BROKENcloud.redhat.com/api/ingress/v1/upload --from-literal=interval=1m -n openshift-config
```
image

### CODE REVIEW

**Comment by gkarat** [comment]
@Fewwy, would be useful if you could also take a look

**Comment by tremes** [comment]
@gkarat I didn't look at the code. I just tried the functionality and it looks good to me. Thank you.

**Review by Fewwy — APPROVED** [review]
LGTM!

**Review by jhadvig — COMMENTED** [review]
@gkarat thanks for the PR! 

From the screenshot I see that the text is muted but I dont see any class added to `StackItem` component. Is that correct?

Screenshot 2022-05-09 at 16 23 59

File: frontend/packages/insights-plugin/src/components/InsightsPopup/mappers.ts
Comment (tremes) [reply frontend/packages/insights-plugin/src/components/InsightsPopup/mappers.ts:50]: Yes

**Comment by gkarat** [comment]
> @gkarat thanks for the PR!
> 
> From the screenshot I see that the text is muted but I dont see any class added to `StackItem` component. Is that correct?
> 
> Screenshot 2022-05-09 at 16 23 59

@jhadvig, yep, this is expected since this is a default styling of the EmptyState https://www.patternfly.org/v4/components/empty-state/#basic

**Comment by jhadvig** [comment]
QE Approver:
 /assign @ikerreyes
Docs Approver: 
/assign @opayne1
PX Approver:
 /assign @RickJWagner

**Review by jhadvig — APPROVED** [review]
/lgtm
/hold
Waiting for all other approvals.

**Pull request merged**
